mp4parse/tests/public.rs

```diff
@@ -309,7 +309,7 @@ fn public_audio_tenc() {
         assert_eq!(a.codec_type, mp4::CodecType::EncryptedAudio);
         match a.protection_info.iter().find(|sinf| sinf.tenc.is_some()) {
             Some(ref p) => {
-                assert_eq!(p.code_name, "mp4a");
+                assert_eq!(p.original_format.value, *b"mp4a");
```

Comment:
It might be a boon to ergonomics to implement equality between `FourCC` and 4-byte arrays:
```rust
impl PartialEq<&[u8; 4]> for FourCC {
    fn eq(&self, other: &&[u8; 4]) -> bool {
        self.value.eq(*other)
    }
}
```
But since we only seem to use this for some tests, I'm fine with leaving this as is.

Reply:
Good call!